src/circuit/gadget/ecc/chip/mul/overflow.rs

```diff
@@ -57,80 +51,49 @@ impl Config {
 
             // Constant expressions
             let one = Expression::Constant(pallas::Base::one());
-            let two_pow_127 = Expression::Constant(pallas::Base::from_u128(1 << 127));
+            let two_pow_124 = Expression::Constant(pallas::Base::from_u128(1 << 124));
+            let two_pow_130 =
+                two_pow_124.clone() * Expression::Constant(pallas::Base::from_u128(1 << 6));
 
             let z_0 = meta.query_advice(self.advices[0], Rotation::prev());
-            let z_127 = meta.query_advice(self.advices[0], Rotation::cur());
+            let z_130 = meta.query_advice(self.advices[0], Rotation::cur());
             let eta = meta.query_advice(self.advices[0], Rotation::next());
 
             let k_254 = meta.query_advice(self.advices[1], Rotation::prev());
             let alpha = meta.query_advice(self.advices[1], Rotation::cur());
 
-            // s_minus_lo_127 = s - sum_{i = 0}^{126} 2^i ⋅ s_i
-            let s_minus_lo_127 = meta.query_advice(self.advices[1], Rotation::next());
+            // s_minus_lo_130 = s - sum_{i = 0}^{130} 2^i ⋅ s_i
```

Comment:
Bits are zero-indexed.

```suggestion
            // s_minus_lo_130 = s - sum_{i = 0}^{129} 2^i ⋅ s_i
```